README.md

```diff
@@ -30,6 +30,8 @@
   
 <b>Notes:</b><br />
 
+- This is a unofficial fork of CrewLink, so any problem, issue or suggestion you have with BetterCrewLink talk to us on our [Discord](https://discord.gg/qDqTzvj4SH) or [GitHub](https://github.com/OhMyGuus/BetterCrewLink/issues), it will not report any problems to Discord nor the official GitHub project of CrewLink they will not support you.
```

Comment:
Should probably read (suggested changes bolded)
"This is **an** unofficial fork of CrewLink, **for** any problem, issue or suggestion you have with BetterCrewLink talk to us on our [Discord](https://discord.gg/qDqTzvj4SH) or [GitHub](https://github.com/OhMyGuus/BetterCrewLink/issues), **do** not report any problems to **the official Discord or GitHub** project of CrewLink **as** they will not support you."

Reply:
Resolved thanks :)